### PR TITLE
Chore: fix minor mistakes

### PR DESCRIPTION
**Pull request by lewinskimaciej**
Fix few random typos, grammar mistakes etc.

### CODE REVIEW

File: pages/service/assets.mdx [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
+Asset management is a crucial part of the Beam integration within your game. Depending on on-chain activities that you want to support in your in-game economy, you have the option to manage assets on behalf of the profiles you created, but also list - buy and make offers on assets through the Beam sdk.
 
+By providing these options through the sdk, you are able to create a frictionless in-game marketplace built on beam without having to worry about creating infrastructure to support it.
 
 ### Roadmap
 
-Throughout the coming months, we'll be releasing additional features that will help you to provide a more complete marketplace experience, with features ranging from programmatic offers, auctions and extensive filtering options to allow you to create a full-fledged in-game marketplace solely by relying on the Beam marketplace.
+Throughout the coming months, we'll be releasing additional features that will help you to provide a more complete marketplace experience, with features ranging from programmatic offers, auctions and extensive filtering options to allow you to create a fully-fledged in-game marketplace solely by relying on the Beam marketplace.
```
Comment (RobbyUitbeijerse) [inline pages/service/assets.mdx]: it's full-fledged

File: pages/service/marketplace/introduction.mdx [hunk 2]
```diff
@@ -21,7 +21,7 @@ In order to create a full marketplace integration it's important to understand a
 
 ### Roadmap
 
-Throughout the coming months, we'll be releasing additional features that will help you to provide a more complete marketplace experience, with features ranging from programmatic offers, auctions and extensive filtering options to allow you to create a full-fledged in-game marketplace solely by relying on the Beam marketplace.
+Throughout the coming months, we'll be releasing additional features that will help you provide a more complete marketplace experience, with features ranging from programmatic offers, auctions and extensive filtering options to allow you to create a fully-fledged in-game marketplace solely by relying on the Beam marketplace.
```
Comment (RobbyUitbeijerse) [inline pages/service/marketplace/introduction.mdx]: it's full-fledged

**Review by RobbyUitbeijerse — APPROVED** [review]

File: pages/service/marketplace/introduction.mdx (hunk 2 above)
Comment (lewinskimaciej) [reply pages/service/marketplace/introduction.mdx]: Yes my lord

File: pages/service/assets.mdx (hunk 1 above)
Comment (lewinskimaciej) [reply pages/service/assets.mdx]: Yes my lord

**Pull request merged**